### PR TITLE
build with github actions - Setup-workflow

### PR DESCRIPTION
**Pull request by tasneem067**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Docker Image CI
```
Comment (mckaymic) [inline .github/workflows/main.yml:1]: Can we change the name of this to something like `Build and test`.  Basically, summarize the steps that are included in this workflow.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: sh
+    steps:
+    - uses: actions/checkout@v2
+    - name: Lint and Test
+      run: |
```
Comment (mckaymic) [inline .github/workflows/main.yml:22]: What is the difference between putting this all sequentially in a single run statement vs separate run statements?

**Review by mckaymic — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 2 above)
Comment (tasneem067) [reply .github/workflows/main.yml:31]: Nothing much.Its just short-hand notation.

**Comment by carrolp** [comment]
Switching to github actions will require additional changes to support multi-arch.

**Pull request closed without merging**